packages/crackql/crackql.install

```diff
@@ -0,0 +1,15 @@
+post_install() {
+  set -e
+  cd /usr/share/crackql
+  python -m virtualenv venv
```

Comment:
Please just use the native venv as per https://github.com/BlackArch/blackarch-pkgbuilds/blob/4ae2d3f269ad76eda2d78f3ea172e9bba6fd1835/python-standalone.install#L4